assets/js/googlesitekit/widgets/components/WidgetRenderer.js

```diff
@@ -24,26 +24,26 @@ import PropTypes from 'prop-types';
 /**
  * WordPress dependencies
  */
-import { Fragment, useMemo } from '@wordpress/element';
+import { Fragment } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
 import { STORE_NAME } from '../datastore/constants';
-import Widget from './Widget';
+import BaseWidget from './Widget';
 import { getWidgetComponentProps } from '../util';
 import { HIDDEN_CLASS } from '../util/constants';
 
 const { useSelect } = Data;
 
 const WidgetRenderer = ( { slug, gridClassName, OverrideComponent } ) => {
 	const widget = useSelect( ( select ) => select( STORE_NAME ).getWidget( slug ) );
-
-	const widgetComponentProps = useMemo( () => getWidgetComponentProps( slug ), [ slug ] );
+	const widgetComponentProps = getWidgetComponentProps( slug );
```

Comment:
Not worth keeping the `useMemo` here?

Reply:
@aaemnnosttv any reason for this?

Reply:
Ah looks like `getWidgetComponentProps` itself now takes care of that using `memize`, so that makes sense. Cleaner approach, so okay to remove here.